lab_Arrays/src/main/java/game/Dor.java

```diff
@@ -0,0 +1,32 @@
+package game;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.util.Random;
+
+public class Dor {
+    private MagicArtifact artifact;
+    private Monster monster;
+
+    public Dor() {
+        int n = new Random().nextInt(2 + 1) + 1;
```

Comment:
all this if else could be extracted to some private method like initRandomDorType() and invoked in constructor